websites/E/EternalNetwork TM/presence.ts

```diff
@@ -138,16 +138,16 @@ presence.on("UpdateData", async () => {
 
 			if (pathname.startsWith("/")) presenceData.state = ttl;
 
-			if (hash.includes("page_ABOUT")) presenceData.state = "About info page !";
+			if (hash.includes("page_ABOUT")) presenceData.state = "About info page";
 
 			if (hash.includes("page_PROGRAMS"))
-				presenceData.state = "Checking radio program !";
+				presenceData.state = "Checking radio program";
 
 			if (hash.includes("page_REQUEST"))
-				presenceData.state = "Requesting a song !";
+				presenceData.state = "Requesting a song";
 
 			if (hash.includes("page_CONTACTS"))
-				presenceData.state = "Contact us page !";
+				presenceData.state = "Using contact form";
```

Comment:
Doesn't sound right

Reply:
Inside the page with the contact form and details I don't have any other idea about what the member can do.

Please, provide me with a message idea of what you think will be good then.

Reply:
The user isn't 100% guaranteed to be actually submitting anything on the form, so you could say `Viewing contact page` - this is also usually what we do for contact pages as other variations of this sound strange

Reply:
Thanks, agree with you I will change it.

Reply:
Changes applied.